sdk/eventgrid/Azure.Messaging.EventGrid/src/Customization/EventGridClient.cs

```diff
@@ -163,5 +195,39 @@ public virtual Response PublishCustomEvents(IEnumerable<object> events, Cancella
                 throw;
             }
         }
+
+        /// <summary>
+        /// Creates a SAS token for use with Event Grid service
+        /// </summary>
+        /// <param name="expirationUtc">Expiration time</param>
+        /// <returns></returns>
```

Comment:
nit: can we fill out the returns value and maybe explain a bit more about what the expiration time means based on the service documentation?